### ISSUE TITLE
[BUG] Centrifuge and Powered Centrifuge allow unintended behaviour in their output slots

### ISSUE THREAD

**Issue by desp4life** [JDKDigital/productive-bees#262]
TLDR: whilst the output slots of a centrifuge or powered centrifuge does not allow items to be placed in it directly, it does allow shift-clicking items in there. This leads to weird (but not gamebreaking) behaviour.

*From here on out, when I mention centrifuge, I mean both the regular, as well as the powered version*

Version: Playing most recent version of All The Mods 7 (0.4.2) which includes version 0.9.0.4 of this mod.

How to reproduce:
1. When the centrifuge is completely empty shift-clicking a compatible block (any comb) will put it into the input slot
2. When the input slot is full and another comb of the same type is shift-clicked, it will be put in the output slots.
3. This is also possible when an incompatable item is shift-clicked (E.G. different type of comb or any other item)
4. The second strange behaviour is when picking up items from the output slots, when not shift-clicking, it allows the picked up itemstack to be larger than the default stack limit (64). Depositing items in the player's inventory when the held itemstack is larger the the limit, does decrease the held stacksize by the stacklimit and deposits an entire stack of items in the inventory.

Expected behaviour:

1. Not being able to input items in the output slots of the centrifuge (in any way, shape or form), unless produced in a production cycle of the item in the input slot. 
2. Items being picked up from the output slots keep their respective stacklimit, and don't allow items to be picked up when the stacklimit is reached.

The following pictures depicts both kinds of strange behaviour I have encountered:
![image](https://user-images.githubusercontent.com/22345306/169666247-93689792-ece4-4019-a7e1-7271bf7fe7a9.png)
![image](https://user-images.githubusercontent.com/22345306/169666410-531c010d-b6f2-4a5c-adcc-e2cdcd63295e.png)

**Comment by JaisDK** [comment]
I'm not gonna do anything about issue 1, it's such a massive pain to deal with UI handling :D

I don't understand how you got a massive stack of wax. Does it stack like that in the output or did you take it out in a specific way?

**Comment by desp4life** [comment]
When multiple stacks are in the output slot (E.G. multiple stacks of wax), I can left-click one stack. This basically makes me hold that stack with my mouse. If I then left-click another stack, it will be added into the same stack that I am holding with my mouse. This can be repeated for multiple stacks in the output slots.
